### ISSUE TITLE
Portal name is output as a list

### ISSUE THREAD

**Issue by tomkroschwald** [telekom/scale#1570]
**Scale Version**
3.0.0-beta.118

**Framework and version**
Vue 3.2.13

**Current Behavior**
The portal name in the header is output to the screen reader as a list.
According to our accessibility report, lists with only one element should be avoided.

You can find the Screenshot here because I couldn't upload it: https://prnt.sc/0AYFcSqtjXaQ

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 109.0.5414.75
